### PR TITLE
Feature/Takes over the address when checking in with several people

### PR DESCRIPTION
**Pull request by devchris**
Closes https://github.com/railslove/recover-backlog/issues/34

I also added the phone number to be copied over. I couldn't think of a scenario where you want to copy the address but then a different phone number. 

Fixes also a tiny html warning that I noticed - thought I put it in here because it's not worth its own PR.
Screen Shot 2020-12-09 at 10 33 43 AM

### CODE REVIEW

File: ui/whitelabels.tsx [hunk 1]
```diff
@@ -68,9 +68,9 @@ const envs = {
           <strong>Bitte speichere die Datei auf deinem Rechner ab.</strong>
         </p>
         <p>
-          <p>
+          <div>
```
Comment (salimhb) [inline ui/whitelabels.tsx:71]: nice catch. Is either of them needed anyway? Wouldn't also work if you drop the tag wrapping `<Warning />`?

Comment (devchris) [reply ui/whitelabels.tsx:71]: It makes a slight styling difference. 

Without the wrapper:
Screen Shot 2020-12-13 at 9 05 58 AM

With the wrapper:
Screen Shot 2020-12-13 at 9 07 15 AM

Putting the wrapper around makes it look exactly how it was before with the `<p>` around. Let me know which one you prefer @salimhb

**Review by salimhb — APPROVED** [review]

**Review by bonflintstone — APPROVED** [review]

**Comment by bonflintstone** [comment]
Is this pr still needed @devchris @salimhb ? Abou the icon wrapper I prefer the second version, in case that is still relevant :)

**Comment by salimhb** [comment]
Yes https://github.com/railslove/recover-backlog/issues/34 is still open.
I think @devchris wasn't aware that a PRs author is responsible for merging after approval.

**Comment by bonflintstone** [comment]
> Yes [railslove/recover-backlog#34](https://github.com/railslove/recover-backlog/issues/34) is still open.
> I think @devchris wasn't aware that a PRs author is responsible for merging after approval.

Perhaps this should be added to the readme then ;) @devchris are you willing to resolve the conflicts and merge this pull request?

**Comment by bitboxer** [comment]
@salimhb I updated this PR and added the missing fields. The HTML fixes have been done in another PR and are no longer needed here. Because of that the PR is now rather small 😉 .

**Pull request merged**